Scripts/Seleccionar.js

```diff
@@ -0,0 +1,12 @@
+/*import { redirigir, objeto } from './Modificar.js';*/
+
+function cargaIframe(Pagina) {
+    const Seleccion = document.querySelectorAll("#root");
+    let ventana = `<iframe src="${Pagina}.html" frameborder="0"></iframe>`;
```

Comment:
Codacy has a fix for the issue: [Replace `··let·ventana·=·`<iframe·src="${Pagina}.html"·frameborder="0"></iframe>`;␍` with `let·ventana·=·`<iframe·src="${Pagina}.html"·frameborder="0"></iframe>`;`](https://app.codacy.com/gh/Jeffer48/Consultarq/pullRequest?prid=9456239)
```suggestion
  let ventana = `<iframe src="${Pagina}.html" frameborder="0"></iframe>`;
```